config/dictionaries/resource.json

```
@@ -438,6 +438,52 @@
       }
     ]
   },
+  {
+    "id": "device_type",
+    "name": "Device type",
+    "fields": [
+      "id",
+      "slug",
+      "name",
+      "is_private",
+      "is_of__cpu_architecture",
+      "belongs_to__device_family"
+    ],
+    "examples": [
+      {
+        "id": "device-type-by-id",
+        "summary": "Get the details of a device type by ID.",
+        "description": "The bearer token is optional for targeting public device types, but is required for private ones.",
+        "method": "GET",
+        "endpoint": "/v6/device_type(<ID>)",
+        "filters": ""
+      },
+      {
+        "id": "device-type-by-handle",
+        "summary": "Get the details of a device type by its SLUG.",
```

```suggestion
        "summary": "Get device type by slug",
```

What slug is that? Where can people find it?